### ISSUE TITLE
Map links in village list possibly off screen

### ISSUE THREAD

**Issue by ormiret** [emfcamp/Website#1713]
On my phone it is partially visible at the edge of the screen and a pain to hit, on smaller phones it's entirely off the right of the screen

**Comment by lukegb** [comment]
Yeah, I added this in hurry during camp and didn't check sizing; I'll try to take a look today.

**Comment by marksteward** [comment]
I think this was fixed by e4d4d9b374c60cf3c8505157f7e9a861c8b3c022? (committed 11 mins after the last comment)
